classes/article/ArticleGalleyDAO.inc.php

```diff
@@ -493,23 +549,26 @@ function getExportable($contextId, $pubIdType = null, $title = null, $author = n
 			FROM	submission_galleys g
 				JOIN submissions s ON (s.submission_id = g.submission_id)
 				LEFT JOIN published_submissions ps ON (ps.submission_id = g.submission_id)
+				JOIN submission_settings ss ON (ss.submission_id = ps.submission_id)
 				JOIN issues i ON (ps.issue_id = i.issue_id)
-				LEFT JOIN submission_files sf ON (g.file_id = sf.file_id)
-				LEFT JOIN submission_files nsf ON (nsf.file_id = g.file_id AND nsf.revision > sf.revision AND nsf.file_id IS NULL )
+				LEFT JOIN submission_galley_files sfg ON (sfg.galley_id = g.galley_id)
+				LEFT JOIN submission_files sf ON (sfg.file_id = sf.file_id)
+				LEFT JOIN submission_files nsf ON (nsf.file_id = sfg.file_id AND nsf.revision > sf.revision AND nsf.file_id IS NULL )
```

Comment:
hmmm... how can these two cases coexist: `nsf.file_id = sfg.file_id AND nsf.file_id IS NULL`?